### DISCUSSION TITLE
Image publisher package removed

### DISCUSSION THREAD

**Discussion by mirellameelo** [OpenVSLAM-Community/openvslam_ros#5] [Ideas]
Hi! I notice the image publisher package was removed.
I took some of [these](https://github.com/OpenVSLAM-Community/openvslam/tree/main/example/util) files from OpenvSLAM and created a ROS2 package (Available [here](https://github.com/mirellameelo/dataset_publisher_ros2)) with the executables: kitti_images, openvslam_video, seeds_images, and euroc_images; and also a Readme file illustrating how to use them. 
I believe it can be a good idea to make it available. In case it is useful, how it could be linked to the project?

**Comment by ymd-stella** [comment]
It is now mentioned at https://openvslam-community.readthedocs.io/en/latest/ros2_package.html#publisher.

**Reply by mirellameelo** [reply]
Nice! Tks. About the readme file for this repository, there is any plans or any task in progress?

**Reply by ymd-stella** [reply]
No.
